package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngtemplate-loader",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Please leave the version as is. I'll update in a separate commit before publishing.